src/App.tsx

```diff
@@ -1,24 +1,145 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 11359;
+
+enum QueryTodos {
+  all = 'All',
+  active = 'Active',
+  completed = 'Completed',
+}
 
 export const App: React.FC = () => {
+  const [myTodos, setMyTodos] = useState<Todo[]>([]);
```

Comment:
Just `todos` will be enough 
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```